ansible.cfg

```diff
@@ -16,6 +16,7 @@ retry_files_save_path = /opt/container_artifact/ansible-retry
 system_warnings = True
 hash_behaviour = merge
 callback_whitelist = profile_tasks
```

Comment:
but why it is still there?

Reply:
It ensures that callbacks still work for older ansible versions